src/FactoryContext.php

```diff
@@ -28,6 +29,7 @@ public function __construct(
         private readonly RecursiveTypeMapperInterface $recursiveTypeMapper,
         private readonly ContainerInterface $container,
         private readonly CacheInterface $cache,
+        private readonly ClassBoundCacheContractFactoryInterface|null $classBoundCacheContractFactory = null,
```

Comment:
Shoud it be added at the end of the list to preserve BC?

Reply:
Good Catch! I added it near to cache for no particular reason (maybe optics?).
Will adjust later.